### PR TITLE
Do type-check to the whole of the app

### PR DESCRIPTION
**Pull request by tk0miya**
Note: This PR contains https://github.com/kaigionrails/conference-app/pull/112.

This gives types to the controllers. And it means types cover the whole of this app!

But this does not mean my work is over. We still have "untyped" types. So I'll keep working on updating types and refactoring  app itself using types.

Thank you for reviewing :-)

### CODE REVIEW

File: sig/handwritten/app/controllers/admin.rbs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module Admin
+end
```
Comment (tk0miya) [inline sig/handwritten/app/controllers/admin.rbs:2]: In Rails app, zeitwerk autoloader automatically defines `Admin` module
internally.  But RBS does not support auto module definition.  Therefore
we need to add the type to the module manually.

File: sig/handwritten/app/controllers/application_controller.rbs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class ApplicationController < ActionController::Base
+  include _RbsRailsPathHelpers
```
Comment (tk0miya) [inline sig/handwritten/app/controllers/application_controller.rbs:2]: To load path helpers inside controllers, we need to include this here.

**Comment by fugakkbn** [comment]
Thank you for your great job!

If I can understand https://github.com/kaigionrails/conference-app/pull/112#discussion_r1384924478 , we can merge this PR.

**Comment by fugakkbn** [comment]
@tk0miya Could you rebase from main branch?

**Comment by tk0miya** [comment]
rebased!

**Review by unasuke — APPROVED** [review]
Wooo~~~

**Comment by tk0miya** [comment]
Wow! Thank you for your support!

**Pull request merged**